phenopacketlab-frontend/src/app/component/phenopacket/phenopacket.component.ts

```diff
@@ -115,54 +131,58 @@ export class PhenopacketComponent implements OnInit, OnDestroy {
     });
   }
 
+  /**
```

Comment:
@iimpulse This is where the phenopacket is updated whenever there is a change in one of its subelements. So if we are logged in, then this is where the phenopacket in the DB is going to be overwritten/re-saved.